fe/src/components/personalityAnalysis/Analysis/QuestionNavigator.tsx

```
@@ -0,0 +1,132 @@
+import { useEffect, useState } from "react";
+import { useRouter } from "next/navigation";
+import {
+  getPersonalityQuestions,
+  postPersonalityAnalyze,
+} from "@/utils/personalityAnalysis";
+import Question from "./Question";
+import ProgressIndicator from "./ProgressIndicator";
+import SplitButton from "@/components/common/Button/SplitButton";
+import { handleApiError } from "@/utils/common/handleApiError";
+
+interface Question {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Question'. Consider to delete it or rename it.

'Question' is defined here:


(lint/suspicious/noRedeclare)